targets/curie_bsp/README.md

```diff
@@ -0,0 +1,120 @@
+###About Curie BSP port
```

Comment:
Just nitpickin': why is the header started at level 3 (`###`) and not at level 1 (`#`)? Also a space is missing (`# About`).